omc3/check_corrections.py

```diff
@@ -0,0 +1,612 @@
+"""
+Correction Test
+---------------
+
+Run a test, i.e. a MAD-X simulation to check how well the correction settings work.
```

Comment:
Since this text is for the documentation, I feel like it wouldn't hurt to give a little description of how the process works here.